src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -190,11 +196,24 @@ def load(
                 data = load_yaml_string(flow_content)
                 content_hash = hash(flow_content)
             is_eager_flow = cls._is_eager_flow(data)
+            is_async_flow = cls._is_async_flow(kwargs)
             if is_eager_flow:
-                return EagerFlow._load(path=flow_path, data=data, raise_error=raise_error, **kwargs)
+                obj = EagerFlow._load(path=flow_path, data=data, raise_error=raise_error, **kwargs)
             else:
                 # TODO: schema validation and warning on unknown fields
-                return ProtectedFlow._load(path=flow_path, dag=data, content_hash=content_hash, **kwargs)
+                obj = ProtectedFlow._load(path=flow_path, dag=data, content_hash=content_hash, **kwargs)
```

Comment:
The `load_flow(async_call=False)` will return `Flow`
But `load_flow(async_call=True)` will return a callable.


flow = load_flow(async_call=True)
flow.xxx = xx

Reply:
Good Catch.